### PR TITLE
update descriptions for image fields

### PR DESCRIPTION
**Pull request by ParryLuke**

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -3130,14 +3130,14 @@ components:
           type: string
           example: image/jpeg
         alt_text:
-          description: Alt text for an image.
+          description: Alt text for an image. Only required and returned if the type is an image.
           type: string
         image_height:
-          description: The height of the image in pixels. Only returned if the type is an image.
+          description: The height of the image in pixels. Only required and returned if the type is an image.
```
Comment (jmorahan) [inline api.swagger.yaml]: Hmm on second thoughts... what does "required" mean here? Don't we have a separate schema for writes, just below? (and it doesn't include the image dimensions, because those are computed from the image)

Comment (ParryLuke) [reply api.swagger.yaml]: required means it has to be in the response when the type of file is an image.

Comment (jmorahan) [reply api.swagger.yaml]: in that case i would rephrase it:
```suggestion
          description: The height of the image in pixels. Returned if and only if the type is an image.
```
